util/CMakeLists.txt

```diff
@@ -14,26 +14,50 @@ endif()
 
 if(BUILD_UTILITIES)
    make_plugin(stdutil "${stdutil_SRCS}" ${MATH_LIBRARY} ${LIBSNDFILE_LIBRARY} ${LIBSNDFILE_SUPPORT_LIBS})
+    target_include_directories(stdutil PRIVATE ${Csound_SOURCE_DIR}/H ${Csound_SOURCE_DIR}/util) # for utilities below
     make_utility(atsa        atsa_main.c)
+    target_include_directories(atsa PRIVATE ${Csound_SOURCE_DIR}/util) # for std_util.h and utilmain.h
     make_utility(csanalyze   csanalyze.c)
     make_utility(cvanal      cvl_main.c)
+    target_include_directories(cvanal PRIVATE
+        ${Csound_SOURCE_DIR}/H # for convolve.h
+        ${Csound_SOURCE_DIR}/util # for std_util.h and utilmain.h
+    )
     make_utility(dnoise      dnoise_main.c)
+    target_include_directories(dnoise PRIVATE ${Csound_SOURCE_DIR}/util) # for std_util.h and utilmain.h
     make_utility(envext      env_main.c)
     make_utility(extractor   xtrc_main.c)
+    target_include_directories(extractor PRIVATE ${Csound_SOURCE_DIR}/util) # for std_util.h and utilmain.h
     make_utility(het_export  hetx_main.c)
+    target_include_directories(het_export PRIVATE ${Csound_SOURCE_DIR}/util) # for std_util.h and utilmain.h
     make_utility(het_import  heti_main.c)
+    target_include_directories(het_import PRIVATE ${Csound_SOURCE_DIR}/util) # for std_util.h and utilmain.h
     make_utility(hetro       het_main.c)
+    target_include_directories(hetro PRIVATE ${Csound_SOURCE_DIR}/util) # for std_util.h, utilmain.h, and SDF/sdif.h
```

Comment:
This seems a bit messy. I'd just add this into the `make_utility` function no? So what if it's not used by all targets, it will save all this extra code.

Reply:
Sure, but they're not used by all utilities, like `csanalyze`. I guess we could make a new `make_std_util` function?

Reply:
Think about it. So what if there are a few extract headers included for a few utilities. We have to maintain this code so write as little of it as possible. Just include the standard csound header locations within `make_utility` and avoid adding all this extra noise in the file.

Reply:
Ok, I added separate `make_internal_utility` and `make_internal_plugin` functions that include internals too